setup-env

```diff
@@ -27,7 +27,7 @@ export EV3DEV_TOOLCHAIN="/opt/arm-2011.03/bin"
 export EV3DEV_ABI="arm-none-eabi-"
 
 # This is where the scripts put all of generated files
-export EV3DEV_BUILD_AREA="$HOME/build-area"
+export EV3DEV_BUILD_AREA="$(pwd)/build-area"
```

Comment:
If we are going to add the `build-area` here though, it needs to be added to the `.gitignore` file. Or we could make the path `$(pwd)/../build-area`.